src/vm/win32threadpool.h

```diff
@@ -372,8 +372,16 @@ class ThreadpoolMgr
 
         } counts;
 
+        // padding to ensure we get our own cache line
+        BYTE padding[64];
+
         Counts GetCleanCounts()
         {
+#ifdef _WIN64
+            // VolatileLoad x64 bit read is atomic
+            return DangerousGetDirtyCounts();
```

Comment:
Due to the order of the barriers in VolitileX `DangerousGetDirtyCounts()` is already
```cpp
result.AsLongLong = counts.AsLongLong;
VOLATILE_MEMORY_BARRIER();
return result;
```
So this should be covered?